### ISSUE TITLE
Tasks: Make it a three-column layout

### ISSUE THREAD

**Issue by ajvsol** [etesync/etesync-web#129]
Currently there are two columns: the filters and the task list.

The filters should be hideable via a harmburger menu icon, which would responsively increase the task list width.

The third column should be the Task Edit screen, instead of it being a separate pop-up. Similarly to Nextcloud Tasks it can slide in from the right when a task is selected:
![](https://raw.githubusercontent.com/nextcloud/tasks/master/screenshots/tasks-1.png)

Also the Tasks app currently has a large margin around it. This should be removed so that the app fills the whole webpage.

**Comment by tasn** [comment]
I agree it should be done (or considered), but there are a few other things I'd like to see first. Like your other tickets and subtasks. :)

**Comment by mnotforme** [comment]
This is such a great idea.
